lib/qualtrics_api/event_subscription.rb

```diff
@@ -10,9 +10,9 @@ class EventSubscription < BaseModel
     end
 
     def delete
-      QualtricsAPI.connection(self)
+      response = QualtricsAPI.connection(self)
         .delete(endpoint)
-        .body["result"]
+      return response.success?
```

Comment:
need some tests for this. 
also I believe if response code is not 200, the gem actually raises an error, see `lib/qualtrics_api/request_error_handler.rb` so i don't think this method will actually return `false`.